lib/phax_machine_sinatra/phax_machine_sinatra.rb

```diff
@@ -50,9 +50,11 @@ class PhaxMachineSinatra < Sinatra::Application
     protected!
     set_phaxio_creds
 
+    files = params['files'].keys.map { |key| params['files'][key]['tempfile'] }.reverse
```

Comment:
Why are we reversing the order of the files?

Reply:
It preserves the order of files the user uploaded